analysis/dataset_clean/fn-ref.R

```diff
@@ -0,0 +1,87 @@
+# Function to set reference levels for factors
+ref <- function(input) {
+  
+  # Handle missing values --------------------------------------------------------
+  print('Handle missing values')
+  
+  input$cov_cat_smoking <- replace(input$cov_cat_smoking, 
```

Comment:
Code positively so replace everything with 'M' that is not a permitted value (E/N/S) for this variable.